packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -211,44 +214,34 @@ export default class InteractSubmode extends Component<Signature> {
           doc.data.meta.realmURL = opts.realmURL.href;
         }
 
+        let cardResource = here.getCard(owner, () => doc, {
+          relativeTo,
+          isAutoSaved: true,
+        });
+        let url = await cardResource.getNewCardURL();
         let newItem = new StackItem({
-          owner: here,
-          newCard: { doc, relativeTo },
+          url,
           format: opts?.cardModeAfterCreation ?? 'edit',
           request: new Deferred(),
           isLinkedCard: opts?.isLinkedCard,
           stackIndex,
         });
-
-        await newItem.ready();
-        if (newItem.cardError) {
-          console.error(
-            `Encountered error creating card:\n${JSON.stringify(
-              doc,
-              null,
-              2,
-            )}\nError: ${JSON.stringify(newItem.cardError, null, 2)}`,
-          );
-          return undefined;
-        }
         here.addToStack(newItem);
-        return newItem.card;
+        return url;
       },
-      viewCard: async (
+      viewCard: (
         cardOrURL: CardDef | URL,
         format: Format = 'isolated',
         opts?: { openCardInRightMostStack?: boolean },
-      ): Promise<void> => {
+      ): void => {
         if (opts?.openCardInRightMostStack) {
           stackIndex = this.stacks.length;
         }
         let newItem = new StackItem({
-          owner: here,
-          url: cardOrURL instanceof URL ? cardOrURL : new URL(cardOrURL.id),
+          url: cardOrURL instanceof URL ? cardOrURL.href : cardOrURL.id,
           format,
           stackIndex,
         });
-        await newItem.ready();
```

Comment:
I don't know how exactly. But, yesterday, the catalog listing had issues where linksTo fields would suddenly disappear when popped onto the stack (after a refresh of the app)

Those links end up as undefined and will be accidentally saved too.

I merged in this PR into my branch and the links are consistently displaying